kvserver: return error properly for GetAggregatedStoreStats

wenyihu6 opened the pull request:
Epic: none
Release note: none

---
**kvserver: pass replication target rq.TransferLease**

Previously, rq.TransferLease was updated to take in replication target (node ID
and store ID) instead of just a store ID, but the log line formatting wasn’t
updated accordingly. This commit fixes the log formatting.

---
**kvserver: return error instead of panic for GetAggregatedStoreStats**

Previously, GetAggregatedStoreStats would panic on error for store.Capacity
instead of returning it. This commit fixes the issue by properly returning the
error back to store.Descriptor.

---
**asim: correctly output rangeID**

Previously, the log labeled a value as a range but printed the storeID. This
commit fixes it to correctly print the rangeID.

wenyihu6 committed:
kvserver: return error properly for GetAggregatedStoreStats

Previously, GetAggregatedStoreStats would panic on error for store.Capacity
instead of returning it. This commit fixes the issue by properly returning the
error back to store.Descriptor.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/152980)
<!-- Reviewable:end -->

wenyihu6 committed:
kvserver: pass replication target rq.TransferLease

Previously, rq.TransferLease was updated to take in replication target (node ID
and store ID) instead of just a store ID, but the log line formatting wasn’t
updated accordingly. This commit fixes the log formatting.

wenyihu6 committed:
asim: correctly output rangeID

Previously, the log labeled a value as a range but printed the storeID. This
commit fixes it to correctly print the rangeID.
